### PR TITLE
Update Set-InboundConnector.md

### PR DESCRIPTION
**Pull request by mohammadahsanafraz**
Add a description for Trusted Organizations:

"TrustedOrganizations" identifies other Office 365 organizations as trusted mail sources for your organization (for example, after acquisitions and mergers). This connector will only work for mail flow between two O365 organizations, so no other parameters will be used.

### CODE REVIEW

File: exchange/exchange-ps/exchange/Set-InboundConnector.md [hunk 1]
```diff
@@ -482,7 +482,7 @@ Accept wildcard characters: False
 ```
 
 ### -TrustedOrganizations
-{{ Fill TrustedOrganizations Description }}
+"TrustedOrganizations" identifies other Office 365 organizations as trusted mail sources for your organization (for example, after acquisitions and mergers). This connector will only work for mail flow between two O365 organizations, so no other parameters will be used.
```
Comment (JohanFreelancer9) [inline exchange/exchange-ps/exchange/Set-InboundConnector.md:485]:
```suggestion
The TrustedOrganizations parameter identifies other Office 365 organizations as trusted mail sources for your organization (for example, after acquisitions and mergers). This parameter only works for mail flow between two Office 365 organizations, so no other parameters are used.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @mohammadahsanafraz, kindly commit the suggested changes. Many thanks.

**Comment by mohammadahsanafraz** [comment]
@microsoft-github-policy-service agree

**Comment by mohammadahsanafraz** [comment]
Sorry guys, is my first time using github and english in not my main language.

@JohanFreelancer9 i couldn't reply earlier, i had a very busy day

**Comment by mohammadahsanafraz** [comment]
i saw that you made changes, so im going to close this pull.

**Pull request closed without merging**